README.md

```diff
@@ -1,4 +1,51 @@
 # certbot
 
-TODO: Enter the cookbook description here.
+Usage
+-----
 
+The certbot cookbook manages SSL certificate generation and renewal within projects.
+
+Add `"recipe[certbot-cdh]"` to enable it.
+
+Remove any SSLs which are stored within data bags.
+
+Finally, include the following in the default attributes:
+
+```json
+"default_attributes": {
+  "certbot": {
+    "cert-owner": {
+      "email": "devops@inviqa.com"
+    }
+  },
+  "nginx": {
+    "shared_config": {
+      "<project-name": {
+        "protocols": ["http", "https"],
+        "includes_first": [
+          "certbot.conf"
+        ]
+      }
+    }
+  }
```

Comment:
this is the usage for certbot-cdh, rather than certbot. Certbot's usage is more like
```
"run_list": {
  "recipe[certbot]"
}

custom receipe: recipes/default.rb:
```
certbot_certonly_webroot 'something' do
   webroot_path '/var/www/certbot'
   email 'devops@example.com'
   domains ['domain1.com', 'domain2.com']
   agree_tos true
end
```